### PR TITLE
Add services per ns cluster limits workload

### PR DESCRIPTION
**Pull request by chaitanyaenr**
This commit will enable us to test the number of services possible
per namespace. The test also supports pushing it by creating max
pssible services per namespace across multiple namespaces.

### CODE REVIEW

**Review by akrzos — COMMENTED** [review]
Everything looks good. Will attempt a quick run before merging.

**Comment by akrzos** [comment]
The test runs successfully but would likely benefit from the concurrent cluster loader once object waits are in place on cluster loader (cc @sjug)

File: workloads/templates/workload-env.yml.j2 [hunk 1]
```diff
@@ -80,4 +80,13 @@ data:
...
   EXPECTED_NAMESPACES_PER_CLUSTER_DURATION: "{{expected_namespaces_per_cluster_duration}}"
+{% elif workload_job == "services-per-namespace" %}
+  PBENCH_INSTRUMENTATION: "{{pbench_instrumentation|bool|lower}}"
+  ENABLE_PBENCH_COPY: "{{enable_pbench_copy|bool|lower}}"
+  SERVICES_PER_NAMESPACE_TEST_PREFIX: "{{services_per_namespace_test_prefix}}"
+  SERVICES_PER_NAMESPACE_CLEANUP: "{{services_per_namespace_cleanup}}"
+  SERVICES_PER_NAMESPACE_BASENAME: "{{services_per_namespace_basename}}"
+  SERVICES_PER_NAMESPACE_PROJECTS: "{{services_per_namespace_projects}}"
+  SERVICES_PER_NAMESPACE_COUNT: "{{services_per_namespace_count}}"
+  EXPECTED_SERVICES_PER_CLUSTER_DURATION: "{{expected_services_per_namespace_duration}}"
```
Comment (akrzos) [inline workloads/templates/workload-env.yml.j2]: I missed this one here:

s/EXPECTED_SERVICES_PER_CLUSTER_DURATION/EXPECTED_SERVICES_PER_NAMESPACE_DURATION

Comment (chaitanyaenr) [reply workloads/templates/workload-env.yml.j2]: @akrzos fixed it, thanks.

**Review by akrzos — APPROVED** [review]
LGTM

**Pull request merged**